src/tests/cli_tests.py

```diff
@@ -636,6 +652,32 @@ def gpg_to_rnp_encryption(filesize, cipher=None, z=None):
     clear_workfiles()
 
 
+def gpg_hidden_to_rnp_encryption(self, filesize, cipher=None, z=None):
+    '''
+    Encrypts with GPG for a hidden recipient and decrypts with RNP
+    '''
+    src, dst, dec = reg_workfiles('cleartext', '.txt', '.gpg', '.rnp')
+    # Generate random file of required size
+    random_text(src, filesize)
+    for armor in [False, True]:
+        # Do for multiple recipients to stress the logic which suppresses decryption error messages
+        for recipient in [KEY_ENCRYPT, 'dummy1@rnp', 'unknown@rnp']:
+            expect_ok = recipient != 'unknown@rnp'
+            # Encrypt cleartext file with GPG
+            gpg_encrypt_file_hidden_to(src, dst, recipient, cipher, z, armor)
+            # Decrypt encrypted file with RNP
+            out, err = rnp_decrypt_file(dst, dec, PASSWORD, expect_ok)
+            if expect_ok:
+                compare_files(src, dec, RNP_DATA_DIFFERS)
+                self.assertEqual(out, '', 'Unexpected stdout contents')
+                err_lines = err.split('\n')
+                err_lines = list(filter(lambda x: x and "premature end of armored input" not in x, err_lines))
```

Comment:
What about adding additional filter for CentOS 7 false-positive? That would allow CI to run.

Reply:
Centos 7 UBSAN false-positive? I think it fails ruby-rnp suite at very least, so any filtering there doesn't change much.

Reply:
Errm, don't see any ruby-rnp failures here in this PR, or do I misunderstand something? There are occassional ruby-rnp failures in s2k tuning, which involve time comparision.